data/json/recipes/recipe_ammo.json

```diff
@@ -1469,7 +1469,7 @@
       [ "solder_wire", 400 ]
     ],
     [
-      [ "cu_pipe", 4 ]
+      [ "copper", 2 ]
```

Comment:
A mistake on my part.Intended to be standardized around half the lead requirement.